.gitlab-ci.yml

```diff
@@ -0,0 +1,150 @@
+default:
```

Comment:
DRY: `interruptible:` can also be moved here
https://docs.gitlab.com/ee/ci/yaml/#default

Reply:
fixed